source/common/http/path_utility.h

```diff
@@ -24,5 +29,21 @@ class PathUtil {
   static absl::string_view removeQueryAndFragment(const absl::string_view path);
 };
 
+using Transformation = std::function<absl::optional<std::string>(absl::string_view)>;
+
+class PathTransformer {
```

Comment:
Can you add comments to the class and its methods?

Reply:
Thanks, will do.